mne_bids_pipeline/steps/preprocessing/_01_data_quality.py

```diff
@@ -232,18 +231,23 @@ def _find_bads_maxwell(
     logger.info(**gen_log_kwargs(message=msg))
 
     if cfg.find_noisy_channels_meg:
-        out_files["auto_scores"] = _auto_scores_path(
-            cfg=cfg,
-            bids_path_in=bids_path_in,
+        out_files["auto_scores"] = bids_path_in.copy().update(
+            suffix="scores",
+            extension=".json",
+            root=cfg.deriv_root,
+            split=None,
+            check=False,
+            session=session,
+            subject=subject,
```

Comment:
Remove `_auto_scores_path` as it's only used here and just be explicit. Note the `subject=subject` which causes it to be written to e.g. `sub-01/` rather than `sub-emptyroom`

Reply:
very good catch, excellent detective work!!!